batch_inference_v2/batch_inference_v2.py

```diff
@@ -215,17 +230,21 @@ def infer(
 
     # Check for performing drift analysis
     if (
-        perform_drift_analysis is None
-        and model_handler._model_artifact.spec.feature_stats is not None
+            perform_drift_analysis is None
+            and model_handler._model_artifact.spec.feature_stats is not None
     ):
         perform_drift_analysis = True
     if perform_drift_analysis:
         context.logger.info("Performing drift analysis...")
         # Get the sample set statistics (either from the sample set or from the statistics logged with the model)
-        sample_set_statistics = mlrun.model_monitoring.api.get_sample_set_statistics(
-            sample_set=model_endpoint_sample_set,
-            model_artifact_feature_stats=model_handler._model_artifact.spec.feature_stats,
-        )
+        statics_input_full_dict = dict(context=context,
```

Comment:
Just curious - why to generate a dictionary and then expend it using `**` into keyword arguments when calling`_get_get_sample_set_statistics_parameters` instead of sending these keyword arguments directly without the dictionary part?

Reply:
you're right, tnx!